src/components/UriViewer.js

```diff
@@ -80,13 +80,13 @@ const PreviewContent = ({ uri, metadata, metadata: { bucket, name }, googleProje
     }
   }
   useOnMount(() => {
-    if (isGs(uri) && isFilePreviewable(metadata)) {
+    if (isFilePreviewable(metadata)) {
       loadPreview()
     }
   })
   return els.cell([
```

Comment:
Not your PR but I was wondering what `els` is, which makes me want to rename the variable to something more explanatory  🤷🏼

Reply:
I'm going to leave this alone … I am also wondering what `els` is supposed to mean so I don't know what to change it to.